Feature

Itswali opened the pull request:
In this PR:
* I create a collection that keep a list of book
* I create a js function to store date in local storage
* I add a function to add book detail in html file as user  enter 
* I add a function to remove the book as user remove.
* I store the detail in localstorage
* I add linters to check for errors

Itswali committed:
add html file with basic form content

Itswali committed:
add javascript file with add or remove book function

Itswali committed:
add linters to check errors

Itswali committed:
Co-authored-by: William Berrocal
Co-authored-by: wiliamberrocal<94255143+williamberrocal@users.noreply.github.com>

Itswali committed:
fix linter errors

Itswali committed:
fix linter errors

Itswali committed:
modified a little

Itswali committed:
documentd correctly

levy002 reviewed (changes requested):
Hello @Itswali 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have a descriptive README, and know that while this is a pair programming project your partner should contribute to this project as well.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

ivanmvh committed:
Update Readme.md file

ivanmvh committed:
Update README.md

ElsonOtake commented on README.md:
```diff
@@ -1 +1,65 @@
+
+- GitHub: [@ivanmvh](https://github.com/ivanmvh)
+- Twitter: [@imprivado](https://twitter.com/imprivado)
+- LinkedIn: [Iván Martínez von Halle] (https://www.linkedin.com/in/ivan-martinez-von-halle/)
```
- [OPTIONAL] I think it's better to remove the space between `]` and `(`. The styling of your link in [markdown language](https://www.markdownguide.org/cheat-sheet/) is not working.

ElsonOtake commented on README.md:
```diff
@@ -1 +1,65 @@
+
+Give a ⭐️ if you like this project!
+
+<!-- ## Acknowledgments
```
- [OPTIONAL] I think removing unused comments from your files is better. I suggest creating the `Acknowledgements` section quoting those you want to thank. Your README will look more professional to those who check your work.

ElsonOtake commented on main.js:
```diff
@@ -0,0 +1,78 @@
+// constant variable declare
```
- [ ] I think it's better to create a collection that keeps a list of books (hint: you can use an array of objects for that). It's a project requirement. You are using the local storage to do this, but the local storage must be used to save your data for the next time you use the app.

ElsonOtake commented on main.js:
```diff
@@ -0,0 +1,78 @@
+    </div>
+    `;
+
+  // adding book with local storage
```
- [ ] Please create a code to add data to the collection that keeps a list of books. It's a project requirement.

ElsonOtake commented on main.js:
```diff
@@ -0,0 +1,78 @@
+    <div> 
+    <p>${title.value}</p>
+    <p>${author.value}</P>
+    <button class ="remove" name="${title.value}">Remove book </button> 
```
- [ ] Please fix the functionality of this button. It's not working.

ElsonOtake commented on main.js:
```diff
@@ -0,0 +1,78 @@
+    item.parentElement.remove();
+    const bookname = item.name;
+
+    // Remove book-info from local storage
```
- [ ] Please create the code to remove the book from the collection. It's a project requirement.

ElsonOtake commented on main.js:
```diff
@@ -0,0 +1,78 @@
+// setup the local storage
+
+if (localStorage.getItem('books') !== null) {
+  const getbook = JSON.parse(localStorage.getItem('books'));
```
- [ ] I think it's better to create a code to present the data from the collection. Load the collection with the data from the local storage when starting. Your code will look more professional and with improved functionality.

ElsonOtake reviewed (changes requested):
STATUS: CHANGES REQUIRED 🟡 

Hi @Itswali @ivanmvh,

Good job so far! 🎉 🎉 
There are some issues that you still need to work on to go to the next project but you are almost there! 🥲 🥲 

![woohoo-youre-almost](https://user-images.githubusercontent.com/77038610/154921145-1f7bb8f6-c843-4ba5-a534-6b840c600558.jpeg)

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️
- PR has a good title and summary. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

ivanmvh committed:
Update README.md

Itswali committed:
makes Changes as per requested

Itswali committed:
fix linter errors

christianbiring1 reviewed (approved):
Hi @Itswali , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
# STATUS: APPROVED :tada: 
![congraaaa](https://user-images.githubusercontent.com/97100091/199574350-b0bf9391-bbdf-4c21-826d-b659837f7870.gif)

## Highlights
- [ ] All linters are green :heavy_check_mark: 
- [ ] Descriptive PR `title` and `summary`
- [ ] Good commit message
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

ivanmvh committed:
Create book.js

Create js folder
Create new class book

ivanmvh committed:
Create class Awesome and methods

Create class Awesome and methods from the existing functions. For each function a method.

Create the method as static then dont need to instance the class with new in order to invoke the methods.

ivanmvh committed:
change index.html to fix path to js folder.
